### PR TITLE
libreoffice: add coreutils, gnugrep in wrapper for some scripts of firejail

### LINKED ISSUE #303980: libreoffice: missing `coreutils` for usage with `firejail`
### Describe the bug
Copied from https://discourse.nixos.org/t/libreoffice-cannot-use-firejail-because-of-path-issue/43289 

configuration.nix
```
programs.firejail.enable = true;
programs.firejail.wrappedBinaries = {
	libreoffice = {
		executable = "${pkgs.lib.getBin pkgs.libreoffice-still}/bin/libreoffice";
		profile = "${pkgs.firejail}/etc/firejail/libreoffice.profile";
	};
};
```
After running libreoffice in terminal, this is some error output:
```
/nix/store/somehash-libreoffice-7.5.9.2/lib/libreoffice/program/soffice: line 51: dirname: command not found
/nix/store/somehash-libreoffice-7.5.9.2/lib/libreoffice/program/soffice: line 145: grep: command not found
/nix/store/somehash-libreoffice-7.5.9.2/lib/libreoffice/program/soffice: line 151: uname: command not found
/nix/store/somehash-libreoffice-7.5.9.2/lib/libreoffice/program/soffice: line 191: /nix/store/someotherhash-libreoffice-7.5.9.2-wrapped/bin/oosplash: No such file or directory
```
I think this is path issue in the soffice script but I do not know how to solve it.

### Additional context
Something similar has been solved for `brave` in https://github.com/NixOS/nixpkgs/pull/258463

### Notify maintainers
(pinging because of recent activity in regard to `libreoffice`)
@LeSuisse @sikmir @7c6f434c @dotlambda 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by JohnRTitor**
## Description of changes

Add coreutils, gnugrep path in wrapper so scripts don't complain
Fixes https://github.com/NixOS/nixpkgs/issues/303980

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by JohnRTitor** [comment]
@Schweber can you test this?

**Comment by JohnRTitor** [comment]
@ofborg eval

**Comment by JohnRTitor** [comment]
@ofborg build libreoffice

**Review by Schweber — APPROVED** [review]

File: pkgs/applications/office/libreoffice/wrapper.nix [hunk 1]
```diff
@@ -32,6 +34,7 @@ let
     "--prefix" "XDG_DATA_DIRS" ":" "${hicolor-icon-theme}/share"
     "--prefix" "GST_PLUGIN_SYSTEM_PATH_1_0" ":"
       "${lib.makeSearchPath "lib/girepository-1.0" unwrapped.gst_packages}"
+    "--suffix" "PATH" ":" "${lib.makeBinPath [ coreutils ]}"
```
[the remark this answers is not in the document]
Comment (JohnRTitor) [reply pkgs/applications/office/libreoffice/wrapper.nix]: Thanks!

**Review by LeSuisse — APPROVED** [review]
Changes looks good, built libreoffice-fresh and it starts as expected.

**Pull request merged**